conan/tools/cmake/cmake.py

```diff
@@ -140,7 +140,19 @@ def install(self, build_type=None):
         if not self._conanfile.should_install:
             return
         mkdir(self._conanfile, self._conanfile.package_folder)
-        self._build(build_type=build_type, target="install")
+
+        bt = build_type or self._conanfile.settings.get_safe("build_type")
+        if not bt:
+            raise ConanException("build_type setting should be defined.")
+        is_multi = is_multi_configuration(self._generator)
+        build_config = "--config {}".format(bt) if bt and is_multi else ""
+
+        pkg_folder = self._conanfile.package_folder.replace("\\", "/")
+        arg_list = ["--install", self._conanfile.build_folder, build_config, "--prefix", pkg_folder]
```

Comment:
Using the ``--install`` feature avoids needing an extra ``cmake.configure()`` in ``package()`` before ``cmake.install()`` to change the ``CMAKE_INSTALL_PREFIX`` that was already defined previously, and this allows ``conan export-pkg`` to work.